### DISCUSSION TITLE
Why is calling the vllm inference model API not recorded in usage?

### DISCUSSION THREAD

**Discussion by Huyueeer** [BerriAI/litellm#4593] [General]
My proxy contains the Qwen model of Azure OpenAI and VLLM inference. I found that the usage of Azure OpenAI will be recorded in the usage, but the usage of VLLM's Qwen will not be recorded in the usage.

**Comment by ishaan-jaff** [comment]
Have you set custom pricing @Huyueeer for vLLM ? 

https://docs.litellm.ai/docs/proxy/custom_pricing#usage-with-openai-proxy-server-1

**Reply by Huyueeer** [reply]
Thanks, I've seen the usage.
